Jellyfin.Plugin.Tvdb/ScheduledTasks/UpdateTask.cs

```diff
@@ -103,8 +104,8 @@ public IEnumerable<TaskTriggerInfo> GetDefaultTriggers()
         private async Task<List<BaseItem>> GetItemsUpdated(CancellationToken cancellationToken)
         {
             double fromTime = DateTimeOffset.UtcNow.AddHours(MetadataUpdateInHours).ToUnixTimeSeconds();
-            IReadOnlyList<EntityUpdate> episodeUpdates = await _tvdbClientManager.GetUpdates(fromTime, cancellationToken, Type.Episodes, Action.Update).ConfigureAwait(false);
```

Comment:
I'd prefer to use IReadonlyList if possible. Should be enough to use toList on the function result

Reply:
Done